CHANGELOG.md

```diff
@@ -2,6 +2,13 @@
 All notable changes to this project will be documented in this file.
 This project adheres to [Semantic Versioning](http://semver.org/).
 
+## [4.7.0](https://github.com/sonata-project/SonataMediaBundle/compare/4.6.0...4.7.0) - 2023-03-09
+### Added
+- [[#2360](https://github.com/sonata-project/SonataMediaBundle/pull/2360)] Added support for `knplabs/gaufrette` ^0.11. ([@jordisala1991](https://github.com/jordisala1991))
+
+### Removed
+- [[#2360](https://github.com/sonata-project/SonataMediaBundle/pull/2360)] Drop support for `doctrine/persistence` ^2.0. ([@jordisala1991](https://github.com/jordisala1991))
```

Comment:
changelog about php 7.4 is missing

Reply:
I forgot. I need to add it manyally since dev-kit pr are ignored...